diff -ru robomongo-shell-9fa2d97e189eac23451f7e565fc46bda86bca690.old/bin/build robomongo-shell-9fa2d97e189eac23451f7e565fc46bda86bca690/bin/build
--- robomongo-shell-9fa2d97e189eac23451f7e565fc46bda86bca690.old/bin/build	2016-12-28 17:25:36.000000000 +0300
+++ robomongo-shell-9fa2d97e189eac23451f7e565fc46bda86bca690/bin/build	2017-04-08 16:37:39.726712638 +0300
@@ -4,14 +4,6 @@
 PROJECT_DIR=$(dirname $BIN_DIR)                                   # Get root directory of project
 source $BIN_DIR/common/setup $1
 
-# Find OpenSSL directory
-OPENSSL_PATH=UNKNOWN
-for i in $(echo $ROBOMONGO_CMAKE_PREFIX_PATH | tr ";" "\n")
- do
-  if [ -f $i/include/openssl/ssl.h ]; then
-   OPENSSL_PATH=$i
-  fi
-done
 if [ $OPENSSL_PATH == UNKNOWN ]; then
  echo -e "Error: Build failed, ROBOMONGO_CMAKE_PREFIX_PATH must contain correct path to openssl. (i.e. /home/user/openssl-1.0.1p_build)"
  exit 1
@@ -19,4 +11,4 @@
 
 # Build mongo
 cd $PROJECT_DIR
-scons mongo --ssl CPPPATH=$OPENSSL_PATH/include LIBPATH=$OPENSSL_PATH/lib -j8 $BUILD_TYPE_SWITCH $MAC_SDK_SWITCH "${@:2}"
+scons mongo --ssl CPPPATH=/usr/include/openssl -j8 $BUILD_TYPE_SWITCH $MAC_SDK_SWITCH "${@:2}"
